cmd/notation/internal/outputs/inspect.go

```diff
@@ -0,0 +1,177 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package outputs
```

Comment:
As discussed last week, can we create a function which takes signature envelope along with additional parameters as input and prints the output?

Reply:
working on this change, will be updated in upcoming rev